### PR TITLE
Add Composite and Composition for OpenSearch

### PR DESCRIPTION
**Pull request by wejdross**
## Checklist

- [ ] PR contains a single logical change (to build a better changelog).
- [ ] Update the documentation.
- [ ] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [ ] Link this PR to related issues or PRs.

### CODE REVIEW

File: packages/tests/sample.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#    backup:
+#      timeOfDay: "12:00:00"
+#    maintenance:
+#      timeOfDay: "12:10:00"
+#      dayOfWeek: "tuesday"
+#    service:
+#      version: "2"
+#      zone: "ch-gva-2"
+#    size:
+#      plan: "startup-4"
```
Comment (glrf) [inline packages/tests/sample.yaml:21]: Could you remove that. We don't have any samples for the other DBaaS and it definitely shouldn't be in the test directory.

**Review by glrf — COMMENTED** [review]
LGTM overall, some smaller things

* Please add the added test cases to github actions here: https://github.com/vshn/component-appcat/blob/master/.github/workflows/packages.yaml#L17-L54
* Please add reference docs for the new packages here: https://github.com/vshn/component-appcat/tree/master/docs/modules/ROOT/pages/references
* Please update the title to something helpful like "Add Composite and Composition for OpenSearch", as the title will show up in the release notes

**Comment by susana-garcia** [comment]
LGTM, I haven't test it locally

**Review by susana-garcia — APPROVED** [review]
apart from the changes that Fabian pointed out, everything looks good

File: packages/tests/sample.yaml (hunk 1 above)
Comment (wejdross) [reply packages/tests/sample.yaml:21]: sure

Comment (glrf) [reply packages/tests/sample.yaml]: Thanks, but please also update the make target as it tries to apply a manifest that doesn't exist anymore

**Pull request merged**